nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/Cluster.java

```diff
@@ -614,6 +615,7 @@ class Decorator implements CreationDecorator<Cluster> {
         public void decorate(AdminCommandContext context, final Cluster instance) throws TransactionFailure, PropertyVetoException {
             Logger logger = ConfigApiLoggerInfo.getLogger();
             LocalStringManagerImpl localStrings = new LocalStringManagerImpl(Cluster.class);
+            Random random = new SecureRandom();
```

Comment:
1. Now `random` gets created even if it's not going to be used.
2. I can not see `random` to be used more than once. Both references seem to be in exclusive branches.

Reply:
The new solution solves this issue -- the static field in a different class is created only if it is used.
Ad used more than once: it's what spotbugs reported, in fact it is called only once, of course.